emulator/src/test/java/com/scalar/client/tool/emulator/EmulatorTerminalTest.java

```diff
@@ -91,4 +91,109 @@ public void input_withSpacesForFlagInput_ShouldReturnProperParamsArrayProperly()
     // Assert
     Assert.assertArrayEquals(paramsArray, mockedParamsArray);
   }
+
+  @Test
+  public void get_WithSpacesForInput_ShouldReturnProperParamsArrayProperly() {
+    // Arrange
+    String input = "get \"key with space\"";
+    String[] mockedParamsArray = { "get", "key with space" };
+
+    // Action
+    String[] paramsArray = emulatorTerminal.paramsParser(input);
+
+    // Assert
+    Assert.assertArrayEquals(paramsArray, mockedParamsArray);
+  }
+
+  @Test
+  public void getJ_WithSpacesForInput_ShouldReturnProperParamsArrayProperly() {
```

Comment:
What is the difference between this test and `get_WithSpacesForInput_ShouldReturnProperParamsArrayProperly`?
Is `-j` option needed?

Reply:
Thank you for detailed review.
Test case updated.